superset-frontend/spec/javascripts/explore/components/FilterBox_spec.jsx

```diff
@@ -58,4 +59,30 @@ describe('FilterBox', () => {
     inst.setState({ selectedValues: { name: 'James' } });
     expect(inst.props.filtersChoices.name.length).toEqual(3);
   });
+
+  it('should support granularity_sqla options', () => {
```

Comment:
added a dumb unit test. Time range filter box will soon be replaced by dashboard component.